lib/src/models/book_images.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:dart_mappable/dart_mappable.dart';
+import 'package:meta/meta.dart';
+
+import '../../../data_model.dart';
+import 'image/image_model.dart';
+
+class BookImagesMapper extends SimpleMapper<BookImages> {
+  const BookImagesMapper();
+
+  @override
+  BookImages decode(dynamic value) {
+    if (value is! Map<String, dynamic>)
+      throw MapperException.unexpectedType(value.runtimeType, BookImages, 'Map<String, dynamic>');
+    
+    final _cover = value['cover'];
+    if (_cover is! Map<String, dynamic>)
+      throw MapperException.unexpectedType(_cover.runtimeType, Cover, 'Map<String, dynamic>');
+    
+    final _thumbnail = value['thumbnail'];
+    if (_thumbnail is! Map<String, dynamic>)
+      throw MapperException.unexpectedType(_thumbnail.runtimeType, CoverThumbnail, 'Map<String, dynamic>');
+  
+    final _pages = value['pages'];
+    if (_pages is! Iterable<dynamic>)
+      throw MapperException.unexpectedType(_pages.runtimeType, List<Image>, 'Iterable<dynamic>');
+    
+    final _media = value['media_id'];
+    if(_media is! String)
+      throw MapperException.unexpectedType(_media.runtimeType, int, 'String');
+
+    final media = int.parse(_media);
+    final cover = Cover(
+      ImageModelMapper.fromMap({
+        ..._cover,
+        'media_id': media,
+        'id': 0,
+      }),
+    );
+
+    final thumbnail = CoverThumbnail(
+      parent: cover,
+      ImageModelMapper.fromMap({
+        ..._cover,
+        'media_id': media,
+        'id': 0,
+      }), 
+    );
```

Comment:
Isn't `cover.thumbnail` enough?

Reply:
It looks literally the same, so probably just make it getter like book.thumbnail => book.cover.thumbnail, and we already have generator for thumbnails via getter AFAIK.